### PR TITLE
GDPR compliance

### LINKED ISSUE #270: GDPR compliance
Update the website **before May 25, 2018** ⏳ to comply with the [General Data Protection Regulation (GDPR)](https://ec.europa.eu/info/law/law-topic/data-protection_en).

- [ ] Let users opt-out and opt-in from being tracked by [Matomo analytics](https://matomo.org).
- [ ] Add a data privacy statement. (check OKFN website for current version)
- [ ] Check if Matomo is configured to mask two bytes of the users IP address.

See discussion in https://openknowledgegermany.slack.com in #gdpr channel. @knutator posted a text template there which might be used.

### PR DESCRIPTION
**Pull request by k-nut**
This should make the website GDPR compliant by

- Removing Piwik tracking (we can always add it back later if needed)
- Adding the Datenschutzerklärung provided by okfn.de and linking to it from the main page

Fixes #270

### CODE REVIEW

**Comment by k-nut** [comment]
@johnjohndoe I think the most important thing here is to bring the website back online. We can add Piwik back in a next PR but for now I think that it is not critical to do so whereas we gain quite a lot by putting the website back out there.

**Comment by johnjohndoe** [comment]
@k-nut I would very much like to bring the website online back again. But since my name is in the imprint I would love to see this happening in alignment with the law. The basic question for me is whether users need to **actively opt-in** before using the website or if an opt-out is fine. By "copying" what bundesregierung.de does I believe we are on the legal side.  
Another option is to replace my name by the OKF in the imprint.

**Comment by stefangrotz** [comment]
I agree with k-nut, we should bring the webside online first and add piwik second. As long as we don't track, there is no problem with GDPR. Every day we wait will cost us users that won't come back after three or four tries.

> Another option is to replace my name by the OKF in the imprint

What do you mean by "to replace my name by the OKF"? If you don't want to support this page anymore we should talk about this in the big group on slack.  Since I am doing a very similar project in my Lab (farmshops.eu) I would offer my Name for the imprint. I have no technical knowledge about this project though, so I think there are better persons to do this, but if it helps to bring the project back online I would do it.

**Comment by k-nut** [comment]
I just talked to okf and they are fine with them being listed in the imprint. So maybe we simply change it to that first?

**Comment by johnjohndoe** [comment]
Thanks @k-nut  :+1: I will take a look.
I manually restarted the Travis CI jobs since they were hanging it seems.

File: impressum.html [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+        <p>pk_ses wird grundsätzlich nach dem Beenden Ihrer Browsersitzung gelöscht, lediglich bei pk_id und pk_ref handelt es sich um ein über eine Browsersitzung hinaus bestehendes (sog. persistentes) Cookie, anhand dessen wir erkennen können, wann ein Nutzer unsere Seite erneut besucht. Sofern Sie das Cookie nicht selbst entfernen, löscht es sich nach 1 Jahr bzw. 6 Monaten selbständig.</p>
+        <p>Die Darstellung unserer Internetseite ist auch ohne Speicherung von Cookies möglich. Sie können das Speichern von Cookies in den Einstellungen ihres Browsers deaktivieren oder diesen so einstellen, dass er Sie über die beabsichtigte Speicherung durch eine Internetseite informiert. In diesem Fall entscheiden Sie über die Annahme des Cookies.</p>
+        <p>Zudem können durch die Entfernung des <a href="https://traffic.okfn.de/index.php?module=CoreAdminHome&action=optOut&language=de">Hakens auf dieser Seite</a> verhindern, dass wir mittels Matomo Analyse-Cookies auf Ihrem Endgerät setzen. In diesem Fall wird ein Cookie gesetzt um uns zu signalisieren, dass Sie der Nutzung widersprochen haben.</p>
```
[the remark this answers is not in the document]
Comment (k-nut) [reply impressum.html]: No, as far as I can tell, you can only opt out of tracking on a Matomo-instance basis.

**Comment by k-nut** [comment]
@johnjohndoe loks good to me. Thanks for fixing the mistakes in there.
I simply forgot the info page but just pushed a new commit that reenables it.

File: impressum.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (johnjohndoe) [reply impressum.html]: Okay. Thank you for the clarification

**Comment by k-nut** [comment]
slightly changed the order of the commits and squashed all Datenschutz related ones into on commit.

File: js/main.js [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 var TILES_URL = '//cartodb-basemaps-a.global.ssl.fastly.net/light_all/{z}/{x}/{y}.png';
 var ATTRIBUTION = '<a id="daten" href="info.html">Über Wo ist Markt?</a> | ' +
-                  '<a id="impressum" href="impressum.html">Impressum</a> | ' +
+                  '<a id="impressum" href="impressum.html">Impressum &amp; Datenschutz</a> | ' +
```
[the remark this answers is not in the document]
Comment (johnjohndoe) [reply js/main.js]: Fix in da63e8365354df104b5d1fdc3045e21dd2b4de68.

File: impressum.html [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+        <h2>Haftung für Links</h2>
+        <p>Unser Angebot enthält Links zu externen Webseiten Dritter, auf deren Inhalte wir keinen Einfluss haben.
+        Deshalb können wir für diese fremden Inhalte auch keine Gewähr übernehmen. Für die Inhalte der verlinkten
+        Seiten ist stets der jeweilige Anbieter oder Betreiber der Seiten verantwortlich. Die verlinkten Seiten
+        wurden zum Zeitpunkt der Verlinkung auf mögliche Rechtsverstöße überprüft. Rechtswidrige Inhalte waren zum
+        Zeitpunkt der Verlinkung nicht erkennbar. Eine permanente inhaltliche Kontrolle der verlinkten Seiten ist
+        jedoch ohne konkrete Anhaltspunkte einer Rechtsverletzung nicht zumutbar. Bei Bekanntwerden von
+        Rechtsverletzungen werden wir derartige Links umgehend entfernen.</p>
+
+        <header>
```
[the remark this answers is not in the document]
Comment (johnjohndoe) [reply impressum.html]: Fixed in 34ee190e85e5f7104f684506e37ba59289da12e4.

File: impressum.html [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+        </ul>
+        <h3>VI. Pflicht zur Bereitstellung von Daten</h3>
+        <p>Sie haben keine vertragliche oder gesetzliche Pflicht uns personenbezogene Daten bereitzustellen. Allerdings sind wir ohne die von Ihnen mitgeteilten Daten nicht in der Lage, Ihnen unsere Services anzubieten.</p>
+        <h3>VII. Bestehen von automatisierten Entscheidungsfindungen (einschließlich Profiling)</h3>
+        <p>Wir verwenden keine automatisierten Entscheidungsfindungen, die Ihnen gegenüber rechtliche Wirkungen entfaltet oder Sie beeinträchtigt.</p>
+        <h3>VIII. Internetspezifische Datenverarbeitungen</h3>
+        <p>Während der Nutzung unserer Internetseiten werden durch uns automatisch technische Zugriffsdaten aufgezeichnet und ausgewertet. Hierfür benutzen wir das Tool "Matomo" (ehemals „Piwik“), ein Open-Source-Programm für Webanalytik, welches statistische Kennwerte zur Nutzung unserer Website auswertet. Diese Daten können jedoch nicht einer bestimmten Person zugeordnet werden; der einzelne Benutzer bleibt hierbei anonym. Weitere Informationen zu "Matomo" finden Sie auf der Website
+            <a href="&gt;https://matomo.org">https://matomo.org</a>.</p>
```
[the remark this answers is not in the document]
Comment (johnjohndoe) [reply impressum.html]: Fixed in 34ee190e85e5f7104f684506e37ba59289da12e4.

File: impressum.html [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+        <p>Die Rechtsgrundlage für die Verwendung von Matomo findet sich in Art. 6 Abs. 1 lit. f) DS-GVO. Unser berechtigtes Interesse besteht darin, dass wir die Nutzung unserer Website zu deren Verbesserung und Optimierung auswerten. Die so erfassten Daten werden für einen Zeitraum von 24 Monaten gespeichert.</p>
+        <h3>IX. Wenn Sie Anmerkungen oder Fragen haben</h3>
+        <p>Wir treffen alle erdenklichen Vorkehrungen zum Schutz und zur Sicherheit Ihrer Daten. Ihre Fragen und Kommentare zum Datenschutz sind uns willkommen, schreiben Sie einfach eine Mail an info@okfn.de.</p>
+        <p>Berlin, Dezember 2018</p>
+        </div>
+
+        <h2>Textquellen</h2>
```
[the remark this answers is not in the document]
Comment (johnjohndoe) [reply impressum.html]: Fixed in 34ee190e85e5f7104f684506e37ba59289da12e4.

**Pull request merged**